extension-impl/dynamic-apollo/src/main/java/com/alipay/sofa/rpc/dynamic/apollo/ApolloDynamicConfigManager.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alipay.sofa.rpc.dynamic.apollo;
+
+import com.alipay.sofa.rpc.auth.AuthRuleGroup;
+import com.alipay.sofa.rpc.dynamic.DynamicConfigKeyHelper;
+import com.alipay.sofa.rpc.dynamic.DynamicConfigManager;
+import com.alipay.sofa.rpc.ext.Extension;
+import com.ctrip.framework.apollo.Config;
+import com.ctrip.framework.apollo.ConfigService;
+
+/**
+ * @author bystander
+ * @version : ApolloDynamicConfigManager.java, v 0.1 2019年06月04日 20:29 bystander Exp $
+ * apollo.cluster need to config or not
+ * app.id need to config
+ */
+
+@Extension(value = "apollo", override = true)
+public class ApolloDynamicConfigManager extends DynamicConfigManager {
+
+    private Config config;
+
+    private String defaultValue = "RPC_DEFAULT_APOLLO_VALUE";
```

Comment:
When users use ApolloDynamicConfigManager but setting noting in Apollo Server, it  will return 'RPC_DEFAULT_APOLLO_VALUE' as the default value.  When get timeout , we expect getting int value instead of 'RPC_DEFAULT_APOLLO_VALUE'

Reply:
ok

Reply:
这个地方就是这么写的。让外层来判断是不是要用这个值，这个地方必须是一个不可能的值